CRITICAL - 2022-01-10 07:35:14 --> Invalid file: template/v_home.php
#0 C:\xampp\htdocs\ppdb-online-ci4\system\View\View.php(198): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('template/v_home...')
#1 C:\xampp\htdocs\ppdb-online-ci4\system\Common.php(1089): CodeIgniter\View\View->render('template/v_home', Array, true)
#2 C:\xampp\htdocs\ppdb-online-ci4\app\Controllers\Home.php(13): view('template/v_home', Array)
#3 C:\xampp\htdocs\ppdb-online-ci4\system\CodeIgniter.php(825): App\Controllers\Home->index()
#4 C:\xampp\htdocs\ppdb-online-ci4\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#5 C:\xampp\htdocs\ppdb-online-ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\ppdb-online-ci4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 C:\xampp\htdocs\ppdb-online-ci4\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#8 {main}
CRITICAL - 2022-01-10 07:35:40 --> Invalid file: template/v_home.php
#0 C:\xampp\htdocs\ppdb-online-ci4\system\View\View.php(198): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('template/v_home...')
#1 C:\xampp\htdocs\ppdb-online-ci4\system\Common.php(1089): CodeIgniter\View\View->render('template/v_home', Array, true)
#2 C:\xampp\htdocs\ppdb-online-ci4\app\Controllers\Home.php(13): view('template/v_home', Array)
#3 C:\xampp\htdocs\ppdb-online-ci4\system\CodeIgniter.php(825): App\Controllers\Home->index()
#4 C:\xampp\htdocs\ppdb-online-ci4\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#5 C:\xampp\htdocs\ppdb-online-ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\ppdb-online-ci4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 C:\xampp\htdocs\ppdb-online-ci4\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#8 {main}
CRITICAL - 2022-01-10 07:36:23 --> Invalid file: template-frontend.php
#0 C:\xampp\htdocs\ppdb-online-ci4\system\View\View.php(198): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('template-fronte...')
#1 C:\xampp\htdocs\ppdb-online-ci4\system\View\View.php(226): CodeIgniter\View\View->render('template-fronte...', Array, true)
#2 C:\xampp\htdocs\ppdb-online-ci4\system\Common.php(1089): CodeIgniter\View\View->render('v_home', Array, true)
#3 C:\xampp\htdocs\ppdb-online-ci4\app\Controllers\Home.php(13): view('v_home', Array)
#4 C:\xampp\htdocs\ppdb-online-ci4\system\CodeIgniter.php(825): App\Controllers\Home->index()
#5 C:\xampp\htdocs\ppdb-online-ci4\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#6 C:\xampp\htdocs\ppdb-online-ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\ppdb-online-ci4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\ppdb-online-ci4\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#9 {main}
CRITICAL - 2022-01-10 10:24:36 --> Call to undefined function form_open()
#0 C:\xampp\htdocs\ppdb-online-ci4\system\View\View.php(210): include()
#1 C:\xampp\htdocs\ppdb-online-ci4\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#2 C:\xampp\htdocs\ppdb-online-ci4\system\Common.php(1089): CodeIgniter\View\View->render('v_login-user', Array, true)
#3 C:\xampp\htdocs\ppdb-online-ci4\app\Controllers\Auth.php(26): view('v_login-user', Array)
#4 C:\xampp\htdocs\ppdb-online-ci4\system\CodeIgniter.php(825): App\Controllers\Auth->login()
#5 C:\xampp\htdocs\ppdb-online-ci4\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#6 C:\xampp\htdocs\ppdb-online-ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\ppdb-online-ci4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\ppdb-online-ci4\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#9 {main}
CRITICAL - 2022-01-10 10:38:17 --> syntax error, unexpected '}'
#0 C:\xampp\htdocs\ppdb-online-ci4\system\Router\Router.php(175): CodeIgniter\Router\Router->autoRoute('auth/login')
#1 C:\xampp\htdocs\ppdb-online-ci4\system\CodeIgniter.php(719): CodeIgniter\Router\Router->handle('auth/login')
#2 C:\xampp\htdocs\ppdb-online-ci4\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->tryToRouteIt(Object(CodeIgniter\Router\RouteCollection))
#3 C:\xampp\htdocs\ppdb-online-ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\ppdb-online-ci4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\ppdb-online-ci4\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2022-01-10 10:40:34 --> Undefined variable: validation
#0 C:\xampp\htdocs\ppdb-online-ci4\app\Views\v_login-user.php(29): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 29, Array)
#1 C:\xampp\htdocs\ppdb-online-ci4\system\View\View.php(210): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\ppdb-online-ci4\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\ppdb-online-ci4\system\Common.php(1089): CodeIgniter\View\View->render('v_login-user', Array, true)
#4 C:\xampp\htdocs\ppdb-online-ci4\app\Controllers\Auth.php(27): view('v_login-user', Array)
#5 C:\xampp\htdocs\ppdb-online-ci4\system\CodeIgniter.php(825): App\Controllers\Auth->login()
#6 C:\xampp\htdocs\ppdb-online-ci4\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#7 C:\xampp\htdocs\ppdb-online-ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ppdb-online-ci4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\ppdb-online-ci4\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2022-01-10 20:09:09 --> Undefined variable: validation
#0 C:\xampp\htdocs\ppdb-online-ci4\app\Views\v_login-user.php(29): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 29, Array)
#1 C:\xampp\htdocs\ppdb-online-ci4\system\View\View.php(210): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\ppdb-online-ci4\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\ppdb-online-ci4\system\Common.php(1089): CodeIgniter\View\View->render('v_login-user', Array, true)
#4 C:\xampp\htdocs\ppdb-online-ci4\app\Controllers\Auth.php(27): view('v_login-user', Array)
#5 C:\xampp\htdocs\ppdb-online-ci4\system\CodeIgniter.php(825): App\Controllers\Auth->login()
#6 C:\xampp\htdocs\ppdb-online-ci4\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#7 C:\xampp\htdocs\ppdb-online-ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ppdb-online-ci4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\ppdb-online-ci4\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2022-01-10 20:11:43 --> Undefined variable: errors
#0 C:\xampp\htdocs\ppdb-online-ci4\app\Views\v_login-user.php(32): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 32, Array)
#1 C:\xampp\htdocs\ppdb-online-ci4\system\View\View.php(210): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\ppdb-online-ci4\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\ppdb-online-ci4\system\Common.php(1089): CodeIgniter\View\View->render('v_login-user', Array, true)
#4 C:\xampp\htdocs\ppdb-online-ci4\app\Controllers\Auth.php(27): view('v_login-user', Array)
#5 C:\xampp\htdocs\ppdb-online-ci4\system\CodeIgniter.php(825): App\Controllers\Auth->login()
#6 C:\xampp\htdocs\ppdb-online-ci4\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#7 C:\xampp\htdocs\ppdb-online-ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ppdb-online-ci4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\ppdb-online-ci4\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2022-01-10 20:12:40 --> Undefined variable: errors
#0 C:\xampp\htdocs\ppdb-online-ci4\app\Views\v_login-user.php(32): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 32, Array)
#1 C:\xampp\htdocs\ppdb-online-ci4\system\View\View.php(210): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\ppdb-online-ci4\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\ppdb-online-ci4\system\Common.php(1089): CodeIgniter\View\View->render('v_login-user', Array, true)
#4 C:\xampp\htdocs\ppdb-online-ci4\app\Controllers\Auth.php(28): view('v_login-user', Array)
#5 C:\xampp\htdocs\ppdb-online-ci4\system\CodeIgniter.php(825): App\Controllers\Auth->login()
#6 C:\xampp\htdocs\ppdb-online-ci4\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#7 C:\xampp\htdocs\ppdb-online-ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ppdb-online-ci4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\ppdb-online-ci4\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2022-01-10 20:12:46 --> Invalid argument supplied for foreach()
#0 C:\xampp\htdocs\ppdb-online-ci4\app\Views\v_login-user.php(32): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Invalid argumen...', 'C:\\xampp\\htdocs...', 32, Array)
#1 C:\xampp\htdocs\ppdb-online-ci4\system\View\View.php(210): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\ppdb-online-ci4\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\ppdb-online-ci4\system\Common.php(1089): CodeIgniter\View\View->render('v_login-user', Array, true)
#4 C:\xampp\htdocs\ppdb-online-ci4\app\Controllers\Auth.php(28): view('v_login-user', Array)
#5 C:\xampp\htdocs\ppdb-online-ci4\system\CodeIgniter.php(825): App\Controllers\Auth->login()
#6 C:\xampp\htdocs\ppdb-online-ci4\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#7 C:\xampp\htdocs\ppdb-online-ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ppdb-online-ci4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\ppdb-online-ci4\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2022-01-10 20:12:49 --> Invalid argument supplied for foreach()
#0 C:\xampp\htdocs\ppdb-online-ci4\app\Views\v_login-user.php(32): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Invalid argumen...', 'C:\\xampp\\htdocs...', 32, Array)
#1 C:\xampp\htdocs\ppdb-online-ci4\system\View\View.php(210): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\ppdb-online-ci4\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\ppdb-online-ci4\system\Common.php(1089): CodeIgniter\View\View->render('v_login-user', Array, true)
#4 C:\xampp\htdocs\ppdb-online-ci4\app\Controllers\Auth.php(28): view('v_login-user', Array)
#5 C:\xampp\htdocs\ppdb-online-ci4\system\CodeIgniter.php(825): App\Controllers\Auth->login()
#6 C:\xampp\htdocs\ppdb-online-ci4\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#7 C:\xampp\htdocs\ppdb-online-ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ppdb-online-ci4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\ppdb-online-ci4\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2022-01-10 20:13:02 --> Invalid argument supplied for foreach()
#0 C:\xampp\htdocs\ppdb-online-ci4\app\Views\v_login-user.php(32): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Invalid argumen...', 'C:\\xampp\\htdocs...', 32, Array)
#1 C:\xampp\htdocs\ppdb-online-ci4\system\View\View.php(210): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\ppdb-online-ci4\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\ppdb-online-ci4\system\Common.php(1089): CodeIgniter\View\View->render('v_login-user', Array, true)
#4 C:\xampp\htdocs\ppdb-online-ci4\app\Controllers\Auth.php(28): view('v_login-user', Array)
#5 C:\xampp\htdocs\ppdb-online-ci4\system\CodeIgniter.php(825): App\Controllers\Auth->login()
#6 C:\xampp\htdocs\ppdb-online-ci4\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#7 C:\xampp\htdocs\ppdb-online-ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ppdb-online-ci4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\ppdb-online-ci4\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2022-01-10 20:15:41 --> Cannot use isset() on the result of an expression (you can use "null !== expression" instead)
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2022-01-10 20:25:48 --> email is not a valid rule.
#0 C:\xampp\htdocs\ppdb-online-ci4\system\Validation\Validation.php(300): CodeIgniter\Validation\Exceptions\ValidationException::forRuleNotFound('email')
#1 C:\xampp\htdocs\ppdb-online-ci4\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules('email', 'E-Mail', 'admin@gmail.com', Array, Array)
#2 C:\xampp\htdocs\ppdb-online-ci4\system\Controller.php(160): CodeIgniter\Validation\Validation->run()
#3 C:\xampp\htdocs\ppdb-online-ci4\app\Controllers\Auth.php(35): CodeIgniter\Controller->validate(Array)
#4 C:\xampp\htdocs\ppdb-online-ci4\system\CodeIgniter.php(825): App\Controllers\Auth->cek_login_user()
#5 C:\xampp\htdocs\ppdb-online-ci4\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#6 C:\xampp\htdocs\ppdb-online-ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\ppdb-online-ci4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\ppdb-online-ci4\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#9 {main}
CRITICAL - 2022-01-10 20:26:52 --> email is not a valid rule.
#0 C:\xampp\htdocs\ppdb-online-ci4\system\Validation\Validation.php(300): CodeIgniter\Validation\Exceptions\ValidationException::forRuleNotFound('email')
#1 C:\xampp\htdocs\ppdb-online-ci4\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules('email', 'E-Mail', 'admin@gmail.com', Array, Array)
#2 C:\xampp\htdocs\ppdb-online-ci4\system\Controller.php(160): CodeIgniter\Validation\Validation->run()
#3 C:\xampp\htdocs\ppdb-online-ci4\app\Controllers\Auth.php(35): CodeIgniter\Controller->validate(Array)
#4 C:\xampp\htdocs\ppdb-online-ci4\system\CodeIgniter.php(825): App\Controllers\Auth->cek_login_user()
#5 C:\xampp\htdocs\ppdb-online-ci4\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#6 C:\xampp\htdocs\ppdb-online-ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\ppdb-online-ci4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\ppdb-online-ci4\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#9 {main}
CRITICAL - 2022-01-10 20:29:50 --> email is not a valid rule.
#0 C:\xampp\htdocs\ppdb-online-ci4\system\Validation\Validation.php(300): CodeIgniter\Validation\Exceptions\ValidationException::forRuleNotFound('email')
#1 C:\xampp\htdocs\ppdb-online-ci4\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules('email', 'E-Mail', 'admin@gmail.com', Array, Array)
#2 C:\xampp\htdocs\ppdb-online-ci4\system\Controller.php(160): CodeIgniter\Validation\Validation->run()
#3 C:\xampp\htdocs\ppdb-online-ci4\app\Controllers\Auth.php(35): CodeIgniter\Controller->validate(Array)
#4 C:\xampp\htdocs\ppdb-online-ci4\system\CodeIgniter.php(825): App\Controllers\Auth->cek_login_user()
#5 C:\xampp\htdocs\ppdb-online-ci4\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#6 C:\xampp\htdocs\ppdb-online-ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\ppdb-online-ci4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\ppdb-online-ci4\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#9 {main}
